assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -32,18 +32,23 @@ import Layout from '../../../../components/layout/layout';
 import { STORE_NAME } from '../../datastore/constants';
 import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
+import { unTrailingSlashIt } from '../../../../util';
 
 const { useSelect } = Data;
 
 const DashboardDetailsWidgetKeywordsTable = () => {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	const referenceSiteURL = useSelect( ( select ) => unTrailingSlashIt( select( CORE_SITE ).getReferenceSiteURL() ) );
 	const footerCTALinkArgs = {
 		resource_id: propertyID,
 		num_of_days: getCurrentDateRangeDayCount(),
 	};
 	if ( url ) {
 		footerCTALinkArgs.page = `!${ url }`;
+	} else if ( isDomainProperty && referenceSiteURL ) {
+		footerCTALinkArgs.page = `*${ referenceSiteURL }`;
```

Comment:
Seems like this needs to be a separate `if` statement, because this `else if` can't ever be truthy as I understand the docs for the `getCurrentEntityURL()` selector.

```suggestion
	}
	
	if ( isDomainProperty && referenceSiteURL ) {
		footerCTALinkArgs.page = `*${ referenceSiteURL }`;
```

It would be worth adding a component test to ensure the CTA link here is correct when `isDomainProperty()` returns `true`.

Reply:
I looked at the IB again and found that we don't even need to add the `page` param in this component.